test/functional/disablewallet.py

```diff
@@ -19,11 +19,6 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
 
```

Comment:
Please add the missing self.extra_args here, and also add an additional test in this file:
```py
assert_raises_jsonrpc(-12, "Keypool ran out", lambda: self.nodes[0].generate(1))
```